jscomp/frontend/bs_builtin_ppx.ml

```diff
@@ -424,15 +424,12 @@ let local_module_name =
     incr v;
     "local_" ^ string_of_int !v
 
-(* Unpack requires core_type package for type inference;
-   use module type bindings and a function to create safe local names instead. *)
-let local_module_type_name =
-  let v = ref 0 in
-  fun ({txt} : Longident.t Location.loc) ->
-    incr v;
-    "__"
-    ^ (Longident.flatten txt |> List.fold_left (fun ll l -> ll ^ l) "")
-    ^ string_of_int !v ^ "__"
+(* Unpack requires core_type package for type inference:
+   Generate a module type name eg. __Belt_List__*)
+let local_module_type_name txt =
+  "_"
```

Comment:
Why single underscore before and double afterwards ?

Reply:
Because I add seperator `"_"` inside `List.fold_left (fun ll l -> ll ^ "_" ^ l) ""`

Reply:
If it is double, then module type name would be `___Belt_List__`

Reply:
Great thanks.